tron/bin/check_tron_jobs.py

```diff
@@ -286,14 +286,17 @@ def guess_realert_every(job):
             return -1
         job_runs = job.get('runs', [])
         job_runs_started = [
-            run for run in job_runs if run['start_time'] is not None
+            run['start_time'] for run in job_runs if run['start_time'] is not None
         ]
+        if len(job_runs_started) == 0 and job.get('run_time', None):
+            job_runs_started = [
+                job['run_time']
```

Comment:
run_time does not exist for a job, it's on each job_run

Reply:
Fixed it. I'm manually testing it rn. Will let you know once it passed.